library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -224,7 +225,7 @@ public void transformPage(View page, float position) {
             ));
             setFirstDayOfWeek(a.getInt(
                     R.styleable.MaterialCalendarView_mcv_firstDayOfWeek,
-                    Calendar.SUNDAY
+                    Calendar.getInstance(Locale.getDefault()).getFirstDayOfWeek()
```

Comment:
I like this, but two things.

1. Can you use `CalendarUtils` to retrieve a Calendar instance, for consistency and future-proofing.
2. Since getting a Calendar instance is more expensive, can we only do so if there was no attribute set? Similar to the left and right arrow mask code above.